reports/README.md

```diff
@@ -0,0 +1,10 @@
+## Authoring krabby reports
```

Comment:
Unless you suspect that all reports will be submitted as PRs to core Krabby this should probably be in a docs folder or in the main README